monkey-runner/build.gradle

```diff
@@ -0,0 +1,24 @@
+task wrapper(type: Wrapper) {
+    gradleVersion = versions.gradleWrapperVersion
+}
+
+buildscript {
+    apply from: 'dependencies.gradle'
+
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath gradlePlugins.gradle
+        classpath gradlePlugins.gradleAndroidCommandPlugin
+        classpath gradlePlugins.bintrayRelease
+    }
+}
+
+apply from: 'dependencies.gradle'
```

Comment:
if you apply at this level you can save the one above in  the`buildscript` closure, just move it above that block